includes/login-register.php

```diff
@@ -159,6 +159,78 @@ function edd_log_user_in( $user_id, $user_login, $user_pass, $remember = false )
 
 }
 
+add_filter( 'wp_login_errors', 'edd_login_register_error_message', 10, 2 );
+/**
+ * Changes the WordPress login confirmation message when using EDD's reset password link.
+ *
+ * @since 2.10
+ * @param object \WP_Error $errors
+ * @param string $redirect
+ * @return void
+ */
+function edd_login_register_error_message( $errors, $redirect ) {
+	$action_is_confirm   = ! empty( $_GET['checkemail'] ) && 'confirm' === sanitize_text_field( $_GET['checkemail'] );
+	$edd_action_is_reset = ! empty( $_GET['edd_reset_password'] ) && 'confirm' === sanitize_text_field( $_GET['checkemail'] );
+	$redirect_url        = ! empty( $_GET['edd_redirect'] ) ? esc_url( $_GET['edd_redirect'] ) : false;
```

Comment:
```suggestion
	$redirect_url        = ! empty( $_GET['edd_redirect'] ) ? $_GET['edd_redirect'] : false;
```

This is being escaped twice -- once here and again further down in the `sprintf()`. I suggest removing this one because it's better to escape later.